src/components/Footer/Footer.module.css

```diff
@@ -38,9 +39,20 @@
 
 .subtitle {
   position: absolute;
-  width: max-content;
   top: 4rem;
   left: 5rem;
+  width: max-content;
```

Comment:
issue: `max-content` for text content results in the problem we where trying to fix and somewhat contradicts `break-word`, see examples at https://developer.mozilla.org/en-US/docs/Web/CSS/max-content

suggestion: we should remove this from the mobile view entirely and only use it for the bigger screen sizes where it does not cause issues